doc/source/user_guide/indexing.rst

```diff
@@ -933,6 +933,20 @@ and :ref:`Advanced Indexing <advanced>` you may select along more than one axis
 
    df2.loc[criterion & (df2['b'] == 'x'), 'b':'c']
 
+``iloc`` distinguishes between different types of boolean indexing.  If the type of boolean indexing is ``Series``,
+an error will be raised. For instance, in the following example, ``df.iloc[sr.values, 1]`` is ok. 
```

Comment:
what type of error?

Reply:
I see below it's a ValueError, just say that here

Reply:
Also you need two backticks around ValueError

Reply:
Done.